growth-ring/Cargo.toml

```diff
@@ -35,4 +35,6 @@ crate-type = ["dylib", "rlib", "staticlib"]
 
 [lints.clippy]
 unwrap_used = "warn"
+indexing_slicing = "warn"
```

Comment:
Maybe do not enable warning on `indexing_slicing` until we have fix most of them, as there are a bunch of places we need to explicitly disable them.

Reply:
Sounds good, not feeling strong about this.